src/core/Object3D.js

```diff
@@ -40,20 +40,24 @@ THREE.Object3D = function () {
 	Object.defineProperties( this, {
 		position: {
 			enumerable: true,
+			configurable: true,
```

Comment:
it is also needed so that one can redefine `position` to make it writable which needed to keep object's positions in synk.